src/audio/src/llext/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Intel Corporation.
```

Comment:
Btw, any reason why this is not mart of the src makefile ?

Reply:
@lgirdwood which one? If you mean https://github.com/thesofproject/sof/blob/main/src/audio/src/CMakeLists.txt then it isn't used in Zephyr builds, (most) audio sources are still built from https://github.com/thesofproject/sof/blob/main/zephyr/CMakeLists.txt Once we change that to use per-component cmake scripts, then yes, I think it should be possible to keep both monolithic and modular instructions in the same file. But I'd have to try to know for sure

Reply:
ok, so we need to circle back to this later when ready.